Remotion/Web/UnitTests/Core/UI/Controls/Rendering/ValidationErrorRendererTest.cs

```diff
@@ -182,7 +212,7 @@ public void RenderValidationError ()
       validationErrorRenderer.RenderValidationErrors (
           _html.Writer,
           "ValidationErrorID",
-          new[] { "ValidationError1", "ValidationError2" });
+          new[] { PlainTextString.CreateFromText ("ValidationError1"), PlainTextString.CreateFromText ("ValidationError2") });
```

Comment:
We should add a line break to the individual text, too, so we can better assert the correct api usage.